src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -757,40 +802,43 @@ EscrowFinish::preclaim(PreclaimContext const& ctx)
 
 template <ValidIssueType T>
 static TER
-escrowFinishApplyHelper(
+escrowUnlockApplyHelper(
     ApplyView& view,
     Rate lockedRate,
     std::shared_ptr<SLE> const& sleDest,
     STAmount const& balance,
     STAmount const& amount,
     AccountID const& issuer,
-    AccountID const& account,
-    AccountID const& dest,
+    AccountID const& uSenderID,
+    AccountID const& uReceiverID,
     bool createAsset,
     beast::Journal journal);
 
 template <>
 TER
-escrowFinishApplyHelper<Issue>(
+escrowUnlockApplyHelper<Issue>(
     ApplyView& view,
     Rate lockedRate,
     std::shared_ptr<SLE> const& sleDest,
     STAmount const& balance,
     STAmount const& amount,
     AccountID const& issuer,
-    AccountID const& account,
-    AccountID const& dest,
+    AccountID const& uSenderID,
```

Comment:
What is `u` stand for? Please don't use Hungarian notation.

Reply:
Either unique or user but I don't know, its used 100's of times in the repo already. If every new developer decides what can and cannot be used we will end up with 500 different nomenclature styes. I prefer to use whatever already exists. `AccountID` is used 1700 times btw.

Reply:
fixed: https://github.com/XRPLF/rippled/pull/5185/commits/239c71985a7b43fd784a2c86ce675d1995efeac0